src/components/layout/index.js

```diff
@@ -79,3 +79,11 @@ export const HeaderActions = styled.div`
   display: flex;
   margin-left: auto;
 `;
+
+export const OpenClosedTag = styled.div`
+  color: ${(props) => props.color};
+  font-size: 14px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 17px;
+`;
```

Comment:
No change required. Let's start using the `Typography` component, which would replace `OpenClosedTag`.